packages/zowe-explorer/__tests__/__unit__/trees/dataset/DatasetTree.unit.test.ts

```diff
@@ -990,9 +989,9 @@ describe("Dataset Tree Unit Tests - Function addFavorite", () => {
 
         await testTree.addFavorite(node);
 
-        expect(testTree.mFavorites[0].label).toBe(`${blockMocks.datasetSessionNode.label}`);
+        expect(testTree.mFavorites[0].label).toBe(`${blockMocks.datasetSessionNode.label?.toString()}`);
         expect(testTree.mFavorites[0].contextValue).toBe(`${Constants.FAV_PROFILE_CONTEXT}`);
-        expect(testTree.mFavorites[0].children[0].label).toBe(`${node.label}`);
+        expect(testTree.mFavorites[0].children[0].label).toBe(`${node.label?.toString()}`);
```

Comment:
Can we remove string interpolation here too (and throughout the rest of the file where possible)?